crates/extension_cli/src/main.rs

```diff
@@ -31,6 +31,28 @@ struct Args {
     scratch_dir: PathBuf,
 }
 
+fn proxy_from_env() -> Option<String> {
```

Comment:
This code seems redundant with that we have here:

https://github.com/zed-industries/zed/blob/d89e2592a13121ad195866ad8c2ca029ed2fcf29/crates/http/src/http.rs#L19-L48

It would be better to find a way to make sure of this existing code instead of duplicating it.

Reply:
I'm not sure if it's worth introducing the entire `http` crate just for one function. However, if you persist, I will proceed with it.